docs/API/core/biometrics.md

```diff
@@ -0,0 +1,38 @@
+# Biometrics
+
+_Biometrics are only available in native apps on supporting devices._
+
+### Check biometrics availability
```

Comment:
Why isn't this a `##` H2 heading instead?

Reply:
Very good question. I'm guessing you did this for stylistic reasons.

https://developers.fliplet.com/API/core/apps.html
https://developers.fliplet.com/API/core/cache.html
https://developers.fliplet.com/API/core/encode.html
https://developers.fliplet.com/API/core/environment.html